CHANGELOG.md

```diff
@@ -9,6 +9,33 @@ You can find and compare releases at the [GitHub release page](https://github.co
 
 ## Unreleased
 
+### Changed
+
+- New signature for `handleFieldBuilder` on `FieldBuilderDirective` interface
+
+```diff
```

Comment:
Let's just put this in UPGRADE.md, I prefer to limit the changelog to single lines